mlir/utils/performance/ck-benchmark-driver/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+find_package(composable_kernel CONFIG 1.0.0 COMPONENTS device_operations)
+if(NOT composable_kernel_FOUND)
+  message(WARNING "Could not find composable-kernels, not building a CK benchmark driver")
+endif()
+
+if (composable_kernel_FOUND)
+  find_package(hip REQUIRED PATHS /opt/rocm)
+
+  set(LLVM_OPTIONAL_SOURCES
+    null.cpp)
+
+  set(LLVM_LINK_COMPONENTS
+    Support)
+  
+  set(LIBS ${composable_kernel_LIBRARY})
+
+  set(EXCLUDE_FROM_ALL TRUE)
+  add_llvm_executable(ck-benchmark-driver
+
+    ck-benchmark-driver.cpp
+    ../common/benchmarkUtils.cpp
```

Comment:
Nit: could I get `benchmarkUtils.cpp` to be its own library? That way we're not compiling it twice.